packages/@groupby/elements-cache-plugin/src/cache-plugin.ts

```diff
@@ -0,0 +1,54 @@
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved
+import { Plugin, PluginMetadata } from '@groupby/elements-core';
+
+/**
+ * The GroupBy Elements cache plugin. This plugin registers an instance of
+ * `Map` that is intended to be used by other plugins as a cache or other data
+ * store. No automatic cache clearing or expiring is performed.
+ */
+export default class CachePlugin implements Plugin {
+  get metadata(): PluginMetadata {
+    return {
+      name: 'cache',
+      depends: [],
+    };
+  }
+
+  /**
+   * The data store in which cache entries are stored.
+   */
+  store: Map<unknown, unknown>;
```

Comment:
We know what the types are for our specific use cases, but since this is a general plugin that other plugins can use, we don't know it in the general case. `unknown, unknown` is likely the most correct type here.